File: SERVER/foody-admin/src/admin/modules/user/restaurant/Restaurant.php
<?php

class Restaurant
{
    private $id;
    private $resCode;
    private $resName;
    private $numberOfBranches;
    private $cityCode;
    private $create_at;
    private $update_at;

    // function Restaurant($id, $resCode, $resName, $resAddress, $imageAddress, $cityCode)
    // {
    //     $this->ID = $id;
    //     $this->ResCode = $resCode;
    //     $this->ResName = $resName;
    //     $this->ResAddress = $resAddress;
    //     $this->ImageAddress = $imageAddress;
    //     $this->CityCode = $cityCode;
    // }

    /**
     * Get the value of id.
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set the value of id.
     *
     * @return self
     */
    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }

    /**
     * Get the value of resCode.
     */
    public function getResCode()
    {
        return $this->resCode;
    }

    /**
     * Set the value of resCode.
     *
     * @return self
     */
    public function setResCode($resCode)
    {
        $this->resCode = $resCode;

        return $this;
    }

    /**
     * Get the value of resName.
     */
    public function getResName()
    {
        return $this->resName;
    }

    /**
     * Set the value of resName.
     *
     * @return self
     */
    public function setResName($resName)
    {
        $this->resName = $resName;

        return $this;
    }

    /**
     * Get the value of cityCode.
     */
    public function getCityCode()
    {
        return $this->cityCode;
    }

    /**
     * Set the value of cityCode.
     *
     * @return self
     */
    public function setCityCode($cityCode)
    {
        $this->cityCode = $cityCode;

        return $this;
    }

    /**
     * Get the value of imageAddress.
     */
    public function getImageAddress()
    {
        return $this->imageAddress;
    }

    /**
     * Set the value of imageAddress.
     *
     * @return self
     */
    public function setImageAddress($imageAddress)
    {
        $this->imageAddress = $imageAddress;

        return $this;
    }

    /**
     * Get the value of create_at.
     */
    public function getCreate_at()
    {
        return $this->create_at;
    }

    /**
     * Set the value of create_at.
     *
     * @return self
     */
    public function setCreate_at($create_at)
    {
        $this->create_at = $create_at;

        return $this;
    }

    /**
     * Get the value of update_at.
     */
    public function getUpdate_at()
    {
        return $this->update_at;
    }

    /**
     * Set the value of update_at.
     *
     * @return self
     */
    public function setUpdate_at($update_at)
    {
        $this->update_at = $update_at;

        return $this;
    }

    /**
     * Get the value of numberOfBranches.
     */
    public function getNumberOfBranches()
    {
        return $this->numberOfBranches;
    }

    /**
     * Set the value of numberOfBranches.
     *
     * @return self
     */
    public function setNumberOfBranches($numberOfBranches)
    {
        $this->numberOfBranches = $numberOfBranches;

        return $this;
    }
}

File: APP/Foody/app/src/main/java/com/gmail/khanhit100896/foody/restaurant/Restaurant.java
package com.gmail.khanhit100896.foody.restaurant;

public class Restaurant {
    private int id;
    private String res_code;
    private String res_name;
    private String num_of_branch;
    private String img_address;
    private String city_code;
    private int action_like;
    private int action_touch;

    public Restaurant(int id, String res_code, String res_name, String num_of_branch, String img_address,
                      String city_code) {
        this.setId(id);
        this.setRes_code(res_code);
        this.setRes_name(res_name);
        this.setNum_of_branch(num_of_branch);
        this.setImg_address(img_address);
        this.setCity_code(city_code);
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getRes_code() {
        return res_code;
    }

    public void setRes_code(String res_code) {
        this.res_code = res_code;
    }

    public String getRes_name() {
        return res_name;
    }

    public void setRes_name(String res_name) {
        this.res_name = res_name;
    }

    public String getNum_of_branch() {
        return num_of_branch;
    }

    public void setNum_of_branch(String num_of_branch) {
        this.num_of_branch = num_of_branch;
    }

    public String getImg_address() {
        return img_address;
    }

    public void setImg_address(String img_address) {
        this.img_address = img_address;
    }

    public String getCity_code() {
        return city_code;
    }

    public void setCity_code(String city_code) {
        this.city_code = city_code;
    }

    public int getAction_like() {
        return action_like;
    }

    public void setAction_like(int action_like) {
        this.action_like = action_like;
    }

    public int getAction_touch() {
        return action_touch;
    }

    public void setAction_touch(int action_touch) {
        this.action_touch = action_touch;
    }
}
